zerver/lib/webhooks/git.py

```diff
@@ -166,3 +181,19 @@ def get_commits_content(commits_data, is_truncated=False):
 def get_short_sha(sha):
     # type: (Text) -> Text
     return sha[:7]
+
+def get_all_committers(commits_data):
+    # type: (List[Dict[str, Any]]) -> List[Tuple[str, int]]
+    committers = defaultdict(int)  # type: Dict[str, int]
+    for commit in commits_data:
+        committers[commit['name']] += 1
+    committers_items = sorted(list(committers.items()), key=lambda item: item[1], reverse=True)  # type: List[Tuple[str, int]]
+    committers_names = [committers_item[0] for committers_item in committers_items]  # type: List[str]
+    committers_values = [committers[name] for name in committers_names]  # type: List[int]
+    if len(committers) > 3:
+        others_no_of_commits = sum(committers_values[3:])
+        for name in committers_names[3:]:
+            committers.pop(name)
+        committers_items = sorted(list(committers.items()), key=lambda item: item[1], reverse=True)
+        committers_items.insert(len(committers_items), ('others', others_no_of_commits))
```

Comment:
isn't enough to do:
`committers_items.append(('others', others_no_of_commits))`?